src/pages/reference/handlers/json-schema.md

```diff
@@ -4,7 +4,11 @@ title: JSON Schema or Samples | API Mesh for Adobe Developer App Builder
 
 # JSON schema handlers
 
-This handler allows you to load any remote REST service, and describe its request/response. With this handler, you can easily customize and control the built GraphQL schema.
+This handler allows you to load any remote REST service, and describe its request/response. With this handler, you can easily customize and control the a GraphQL schema in API Mesh for Adobe Developer App Builder.
+
+<InlineAlert variant="info" slots="text"/>
+
+If your source handler's schema is modified, you must [update your mesh](../../gateway/create-mesh.md#update-an-existing-mesh) to allow API Mesh to cache any changes.
```

Comment:
`If your REST service request / response format is modified, you have to update your mesh config with modified request / response and you must [update your mesh](../../gateway/create-mesh.md#update-an-existing-mesh) to allow API Mesh to cache any changes.`

We can modify here to more specific for JSON schema